app/components/Form/FormBase.tsx

```diff
@@ -39,10 +39,10 @@ const FormBase: React.FC<Props> = ({
       schema
     );
     onFormErrors(makeErrorsObject(errorSchema));
-  }, []);
+  }, [formData, makeErrorsObject, onFormErrors, formData]);
```

Comment:
Adding these dependencies causes infinite re-renders. Despite the eslint warning, I think the empty array is correct. We want this useEffect() to essentially work like ComponentDidMount() and not depend on the makeErrorsObject / onFormErrors functions. 
https://stackoverflow.com/questions/58579426/in-useeffect-whats-the-difference-between-providing-no-dependency-array-and-an

Reply:
pretty sure we don't want errors on mount anyway, so this effect could be deleted?

Reply:
removed the error checking on form load so this is no longer an issue